.gitattributes

```diff
@@ -1 +1 @@
-* text=auto
+* text=auto eol=lf
```

Comment:
I noticed a lot of error squiggles in my editor (on Windows) were due to ESLint wanting LF, but git defaulting to CRLF on checkout. This should help them agree on things

Reply:
Awesome. :) Good call!